mlos_bench/mlos_bench/tests/storage/sql/fixtures.py

```diff
@@ -99,7 +118,26 @@ def exp_storage_with_trials(exp_storage: SqlStorage.Experiment) -> SqlStorage.Ex
                 # And some influence from the tunable value.
                 "score": tunable_value_norm + random() / 100
             })
-    return exp_storage
+    return exp
+
+
+@pytest.fixture
+def exp_storage_with_trials(exp_storage: SqlStorage.Experiment) -> SqlStorage.Experiment:
+    # pylint: disable=too-many-locals
```

Comment:
```suggestion
```
definitely don't need this one

Reply:
Was complaining for me locally in VSCode earlier, but yeah not on the CLI.